spec/validators/fee/interim_fee_validator_spec.rb

```diff
@@ -164,57 +164,16 @@
     end
   end
 
-  describe 'common warrant fee validations' do
-    # TODO: share these with warrant fee validator spec
+  describe 'interim warrant fee validations' do
     let(:fee) { interim_warrant_fee }
 
+    include_examples 'common warrant fee validations'
+
     describe '#validate_warrant_issued_date' do
       it 'should be valid if present and in the past' do
         fee.warrant_issued_date = Date.today
         expect(fee).to be_valid
       end
```

Comment:
why is this not shareable? -  i don't know

Reply:
@jsugarman I don't think it can be shared because the validations defined for warrant_issued_date are different in interim_fee_validator.rb (https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/validators/fee/interim_fee_validator.rb) and warrant_fee_validator.rb (https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/validators/fee/warrant_fee_validator.rb) specifically here https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/e3c7fa094fa0a3a0c9450ab677583ad64ea3b64e/app/validators/fee/warrant_fee_validator.rb#L9